stable/traefik/values.yaml

```diff
@@ -52,6 +52,7 @@ ssl:
   enforced: false
   upstream: false
   insecureSkipVerify: false
+  cipherSuites: []
```

Comment:
Can you comment this out?

Reply:
yup, sure I can. But does this change anything? Basically the explanation in the README is that I've managed to come up with. Fell free to change it. Imo it should be clear enough.
PS. I've made the new commit commenting the above out.